### PR TITLE
Updated outputs for namespace [m], [l], [k], [i]

### PR DESCRIPTION
**Pull request by MrMCake**
# Change

- Updated outputs for namespace 
  - [m], 
  - [l], 
  - [k], 
  - [i]
  
Updated dependency pipelines. Reference
[.Platform: Dependencies](https://github.com/Azure/ResourceModules/actions/workflows/platform.dependencies.yml)

Azure DevOps [reference](https://dev.azure.com/servicescode/infra-as-code-source/_build/results?buildId=154316&view=results)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

### CODE REVIEW

File: arm/Microsoft.Insights/actionGroups/readme.md [hunk 1]
```diff
@@ -118,11 +118,11 @@ Tag names and tag values can be provided as needed. A tag can be left without a
...
-| `actionGroupResourceId` | string | The resource ID of the action group  |
+| `name` | string | The name of the action group  |
+| `resourceGroupName` | string | The resource group the action group was deployed into |
+| `resourceId` | string | The resource ID of the action group  |
 
 ## Template references
 
 - [Actiongroups](https://docs.microsoft.com/en-us/azure/templates/microsoft.insights/2019-06-01/actionGroups)
-- [Roleassignments](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2021-04-01-preview/roleAssignments)
+- [Roleassignments](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/roleAssignments)
```
Comment (eriqua) [inline arm/Microsoft.Insights/actionGroups/readme.md:128]: Here as well for the link pointing to the correct API version

File: arm/Microsoft.ManagedServices/registrationDefinitions/deploy.bicep [hunk 2]
```diff
@@ -44,13 +44,13 @@ module registrationAssignment_rg '.bicep/nested_registrationAssignment.bicep' =
...
-output registrationDefinitionResourceId string = registrationDefinition.id
+output resourceId string = registrationDefinition.id
 
 @description('The subscription the registration definition was deployed into')
-output registrationDefinitionSubscription string = subscription().displayName
+output subscriptionName string = subscription().displayName
 
 @description('The registration assignment resource ID')
-output registrationAssignmentId string = empty(resourceGroupName) ? registrationAssignment_sub.id : registrationAssignment_rg.outputs.registrationAssignmentId
+output registrationAssignmentId string = empty(resourceGroupName) ? registrationAssignment_sub.id : registrationAssignment_rg.outputs.resourceId
```
Comment (eriqua) [inline arm/Microsoft.ManagedServices/registrationDefinitions/deploy.bicep:56]:
```suggestion
output assignmentResourceId string = empty(resourceGroupName) ? registrationAssignment_sub.id : registrationAssignment_rg.outputs.resourceId
```

File: arm/Microsoft.ManagedServices/registrationDefinitions/readme.md [hunk 3]
```diff
@@ -59,10 +59,10 @@ remote/managing tenant.
 
 | Output Name | Type | Description |
 | :-- | :-- | :-- |
+| `name` | string | The name of the registration definition |
 | `registrationAssignmentId` | string | The registration assignment resource ID |
```
Comment (eriqua) [inline arm/Microsoft.ManagedServices/registrationDefinitions/readme.md:63]:
```suggestion
| `assignmentResourceId` | string | The registration assignment resource ID |
```

File: arm/Microsoft.Insights/actionGroups/readme.md (hunk 1 above)
Comment (MrMCake) [reply arm/Microsoft.Insights/actionGroups/readme.md:128]: See other comments

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**